tests/providers/amazon/aws/sensors/test_rds.py

```diff
@@ -161,6 +143,7 @@ def test_db_instance_snapshot_poke_true(self):
 
     @mock_rds
     def test_db_instance_snapshot_poke_false(self):
+        _create_db_instance(self.hook)
```

Comment:
This sensor checks the existence of a snapshot associated with a db instance, but the unit tests didn't create the db instance.